# Chapter 9.2

**4. Consider again the conditions of Exercise 2, but suppose now that it is desired to find a test procedure for which α(δ) ≤ 0.1 and β(δ) is a minimum.**

From exercise 2, we have:

$$
f_0(x) = \begin{cases}
1 & \text{if } x \in [0, 1] \\
0 & \text{otherwise}
\end{cases}
$$

$$
f_1(x) = \begin{cases}
2x & \text{if } x \in [0, 1] \\
0 & \text{otherwise}
\end{cases}
$$

$$H_0: f (x)= f_0(x)$$
$$H_1: f (x)= f_1(x)$$

**a. Describe the procedure.**

Both $f_0$ and $f_1$ describe simply hypotheses. So, we can apply the Neyman-Pearson lemma.

The most powerful test with $\alpha = 0.1$ is given by,

$$
{f_1(x) \over f_0(x)} \geq k
$$

We need to solve for $k$ such that,

$$
P\left({f_1(x) \over f_0(x)} \geq k  \big| H_0 \right) = 0.1
$$



Under both hypotheses, the support of $x$ is $[0, 1]$. So, we can assume that this condition holds.

Consider the event,

$$
{f_1(x) \over f_0(x)} \ge k
$$

This is equivalent to,

$$
{2x \over 1} \ge k
$$

$$
2x \ge k
$$

Under the null, the probability of this event is given by,

$$
P(x \ge k / 2 | H_0) = \int_{k/2}^1 f_0(x) dx
$$

$$
P(x \ge k / 2 | H_0) = \int_{k/2}^1 dx
$$

$$
P(x \ge k / 2 | H_0) = \big[x\big]_{k/2}^1
$$

$$
P(x \ge k / 2 | H_0) = 1 - k/2
$$

So,

$$
\alpha = 1 - k/2
$$

$$
0.1 = 1 - k/2
$$

$$
k/2 = 0.9
$$

$$
k = 1.8
$$

So, under our test procedure, we reject if,

$$
2x \gt 1.8
$$

Equivalently,

$$
x \gt 0.9
$$

**b. Determine the minimum value of β(δ) attained by the procedure.**

We have that, under the null, $x$ is from the uniform on $[0, 1]$. So,

$$
P(x \ge 0.9 | H_0) = 0.1
$$

Meanwhile,

$$
P(x \ge 0.9 | H_1) = \int_{0.9}^1 f_1(x) dx
$$

So,

$$
P(x \ge 0.9 | H_1) = \int_{0.9}^1 2x dx
$$

$$
P(x \ge 0.9 | H_1) = \left[ {x^2 \over 2}\right]_{0.9}^1 2x
$$

$$
P(x \ge 0.9 | H_1) = {1 \over 2} - {0.9^2 \over 2}
$$

$$
P(x \ge 0.9 | H_1) = {1 - 0.81 \over 2}
$$

$$
P(x \ge 0.9 | H_1) = {0.19 \over 2}
$$

$$
P(x \ge 0.9 | H_1) = 0.095
$$

So, the true positive rate is $0.095$. This seems pretty poor, but keep in mind that we only have one data point, and the distributions are fairly similar.

**7. Suppose that $X_1, . . . , X_n$ form a random sample from the normal distribution with known mean $μ$ and unknown variance $σ^2$, and the following simple hypotheses are to be tested:**

$$H_0: σ^2= 2$$
$$H_1: σ^2 = 3$$

**a. Show that among all test procedures for which $α(δ) ≤ 0.05$, the value of $β(δ)$ is minimized by a procedure that rejects $H_0$ when $\sum_{i = 1}^n (X_i - \mu)^2 \gt c$.**

Let $L_0$ denote the likelihood of $X$ under the null and $L_1$ denote the likelihood under the alternate.

Since both the null and alternate are simple hypotheses, we can apply the Neyman-Pearson lemma. So, for any test with $\alpha \le 0.05$, the most powerful test is given by,

$$
\delta : {L_1 \over L_0} \gt k
$$

We have that,

$$
L_0 = \prod_{i = 1}^n \frac{1}{\sqrt{2\pi(2)}} \exp\left(-\frac{(x_i-\mu)^2}{2(2)}\right)
$$

$$
L_1 = \prod_{i = 1}^n \frac{1}{\sqrt{2\pi(3)}} \exp\left(-\frac{(x_i-\mu)^2}{2(3)}\right)
$$

Then,

$$
L_0 = \left(\frac{1}{\sqrt{2\pi(2)}}\right)^n \prod_{i = 1}^n  \exp\left(-\frac{(x_i-\mu)^2}{2(2)}\right)
$$

$$
L_1 = \left(\frac{1}{\sqrt{2\pi(3)}} \right)^n \prod_{i = 1}^n \exp\left(-\frac{(x_i-\mu)^2}{2(3)}\right)
$$

Then,

$$
L_0 = \left(\frac{1}{\sqrt{2\pi}\sqrt{2}}\right)^n \exp\left( \sum_{i = 1}^n -\frac{(x_i-\mu)^2}{2(2)}\right)
$$

$$
L_1 = \left(\frac{1}{\sqrt{2\pi}\sqrt{3}} \right)^n \exp\left( \sum_{i = 1}^n -\frac{(x_i-\mu)^2}{2(3)}\right)
$$

Then,

$$
L_0 =
\left(\frac{1}{\sqrt{2\pi}}\right)^n
\left(\frac{1}{\sqrt{2}}\right)^n
\exp \left( {1 \over 4} \sum_{i = 1}^n -(x_i-\mu)^2 \right)
$$

$$
L_1 =
\left(\frac{1}{\sqrt{2\pi}} \right)^n
\left(\frac{1}{\sqrt{3}} \right)^n
\exp \left( {1 \over 6} \sum_{i = 1}^n -(x_i-\mu)^2 \right)
$$

Let $\phi$ denote the likelihood ratio, so,

$$
\phi = {L_1 \over L_0}
$$

Then,

$$
\phi = {
    {
        \left(\frac{1}{\sqrt{2\pi}}\right)^n
        \left(\frac{1}{\sqrt{2}}\right)^n
        \exp \left( {1 \over 4} \sum_{i = 1}^n -(x_i-\mu)^2 \right)
    } \over
    {
        \left(\frac{1}{\sqrt{2\pi}} \right)^n
        \left(\frac{1}{\sqrt{3}} \right)^n
        \exp \left( {1 \over 6} \sum_{i = 1}^n -(x_i-\mu)^2 \right)
    }
}
$$

$$
\phi =
\left(\frac{\sqrt{3}}{\sqrt{2}}\right)^n
\exp \left(
    {1 \over 4} \sum_{i = 1}^n -(x_i-\mu)^2 -
    {1 \over 6} \sum_{i = 1}^n -(x_i-\mu)^2
\right)
$$

$$
\phi =
\left(\frac{\sqrt{3}}{\sqrt{2}}\right)^n
\exp \left(
\left(
    {1 \over 4}-
    {1 \over 6}
\right)
\sum_{i = 1}^n -(x_i-\mu)^2
\right)
$$

$$
\phi =
\left(\frac{\sqrt{3}}{\sqrt{2}}\right)^n
\exp \left(
{1 \over 12}
\sum_{i = 1}^n -(x_i-\mu)^2
\right)
$$

Consider the natural logarithm,

$$
\ln(\phi) =
n \ln \left(\frac{\sqrt{3}}{\sqrt{2}}\right) +
{1 \over 12}
\sum_{i = 1}^n -(x_i-\mu)^2
$$

We are interested in the event where,

$$
\delta: \phi \gt k
$$

But, $\phi$ is a ratio of two even quantities, so $\phi$ is positive. So, this is equivalent to the event where,

$$
\delta: \ln(\phi) \gt \ln(k)
$$

Then,

$$
\delta:
n \ln \left(\frac{\sqrt{3}}{\sqrt{2}}\right) +
{1 \over 12}
\sum_{i = 1}^n -(x_i-\mu)^2
\gt
\ln(k)
$$

$$
\delta:
{1 \over 12}
\sum_{i = 1}^n -(x_i-\mu)^2
\gt
\ln(k) - n \ln \left(\frac{\sqrt{3}}{\sqrt{2}}\right)
$$

$$
\delta:
\sum_{i = 1}^n (x_i-\mu)^2
\gt
-12 \left(\ln(k) - n \ln \left(\frac{\sqrt{3}}{\sqrt{2}}\right)\right)
$$

But, the right hand side of this equation is a constant. So, for upper bounded $\alpha$, for some $c \in \mathbb{R}$, the most powerful test is given by,

$$
\delta:
\sum_{i = 1}^n (x_i-\mu)^2
\gt
c
$$

**b. For n = 8, find the value of the constant c that appears
in part (a).**

We need to find $c$ such that,

$$
P \left(\sum_{i = 1}^n (X_i - \mu)^2 \gt c \bigg| H_0 \right) \le 0.05
$$

Under the null, Each $X_i$ is normally distributed,

$$
X_i | H_0 \sim \text{Normal}(\mu, 2)
$$

Subtract ${X_i - \mu \over \sqrt{2}}$. This is a linear transformation of a normal, so it's normal.

$$
E\left[ {X_i - \mu \over \sqrt{2}} \right] = {1 \over \sqrt{2}}(E[ X_i] - \mu) = 0
$$

By the properties of variance,

$$
var\left[ {X_i - \mu \over \sqrt{2}} \right] = {1 \over 2}var[X_i] = 1
$$

So,

$$
{X_i - \mu \over \sqrt{2}} \bigg| H_0 \sim \text{Normal}(0, 1)
$$

So,

$$
{(X_i - \mu)^2 \over 2} \bigg| H_0 \sim \chi^2(1)
$$

So,

$$
{1 \over 2}\sum_{i = 1}^n (X_i - \mu)^2 \bigg| H_0 \sim \chi^2(n)
$$

We are interested in the event where,

$$
\sum_{i = 1}^n (X_i - \mu)^2 \gt c
$$

This is equivalent to,

$$
{1 \over 2} \sum_{i = 1}^n (X_i - \mu)^2 \gt {c \over 2}
$$

Let $\Phi_n$ denote the cdf of the $\chi^2$ distribution with $n$ degrees of freedom. Then,

$$
P\left({1 \over 2} \sum_{i = 1}^n (X_i - \mu)^2 \gt {c \over 2}\right) = 1 - \Phi_n(c/2)
$$

So,

$$
P\left(\sum_{i = 1}^n (X_i - \mu)^2 \gt c\right) = 1 - \Phi_n(c/2)
$$

So,

$$
\alpha(\delta) = 1 - \Phi_n(c/2)
$$

Suppose,

$$
\alpha(\delta) = 0.05
$$

Then,

$$
1 - \Phi_n(c/2) = 0.05
$$

$$
0.95 = \Phi_n(c/2)
$$

Let $\Phi_n^{-1}$ denote the inverse of $\Phi_n$. Then,

$$
{c \over 2} = \Phi_n^{-1}(0.95)
$$

$$
c = 2\Phi_n^{-1}(0.95)
$$

This can be solved numerically for $n = 8$

In [1]:
from scipy import stats

c = 2 * stats.chi2.ppf(0.95, df = 8)
c

np.float64(31.0146261117309)

So,

$$
c = 31.0146261117309
$$

So, our test is,

$$
\sum_{i = 1}^n (X_i - \mu)^2 \gt 31.0146261117309
$$

# Section 9.3

**2. Suppose that $X_1, . . . , X_n$ form a random sample from the normal distribution with known mean $μ$ and unknown variance $σ^2$ ($σ^2 > 0$). Show that the joint p.d.f. of $X_1, . . . , X_n$ has a monotone likelihood ratio in the statistic $\sum_{i = 1}^n(X_i - \mu)^2$.**

Let,

$$
T = \sum_{i = 1}^n (X_i - \mu)^2
$$

We have that,

$$
P(X_1...X_n = x_1...x_n | \sigma^2) = \prod_{i = 1}^n (2\pi \sigma^2)^{-1/2} \exp\left( -{(x_i - \mu)^2 \over 2\sigma^2} \right)
$$

So, the log likelihood is given by,

$$
\mathcal{L}(\sigma^2) = \sum_{i = 1}^n -{1 \over 2} \left( \ln(2\pi) + \ln(\sigma^2) + {(X_i - \mu)^2 \over \sigma^2} \right)
$$

$$
\mathcal{L}(\sigma^2) =  -{1 \over 2} \left( n\ln(2\pi) + n\ln(\sigma^2) + \sum_{i = 1}^n{(X_i - \mu)^2 \over \sigma^2} \right)
$$

Let,

$$
\phi = \mathcal{L}(\sigma_1^2) - \mathcal{L}(\sigma_0^2)
$$

Then,

$$
\phi = {
     -{1 \over 2} \left( n\ln(2\pi) + n\ln(\sigma_1^2) + \sum_{i = 1}^n{(X_i - \mu)^2 \over \sigma_1^2} \right)
     -{1 \over 2} \left( n\ln(2\pi) + n\ln(\sigma_0^2) + \sum_{i = 1}^n{(X_i - \mu)^2 \over \sigma_0^2} \right)
}
$$

$$
\phi = {
     -{1 \over 2} \left( n\ln(\sigma_1^2) + \sum_{i = 1}^n{(X_i - \mu)^2 \over \sigma_1^2}
     - n\ln(\sigma_0^2) - \sum_{i = 1}^n{(X_i - \mu)^2 \over \sigma_0^2} \right)
}
$$

$$
\phi = {
    -{1 \over 2} \left(
        n\ln(\sigma_1^2) - n\ln(\sigma_0^2)
        + {1 \over \sigma_1^2} \sum_{i = 1}^n(X_i - \mu)^2
        - {1 \over \sigma_0^2} \sum_{i = 1}^n(X_i - \mu)^2
    \right)
}
$$

Let,

$$
T = \sum_{i = 1}^n(X_i - \mu)^2
$$

Then,

$$
\phi = {
    -{1 \over 2} \left(
        n\ln(\sigma_1^2) - n\ln(\sigma_0^2)
        + \left({1 \over \sigma_1^2} - {1 \over \sigma_0^2}\right)
        T
    \right)
}
$$

Then,

$$
{d \over dT}\phi = {
    -{1 \over 2} \left({1 \over \sigma_1^2} - {1 \over \sigma_0^2}\right)
}
$$

So, the derivative of the log likelihood ratio with respect to $T$ is a constant. Under the assumption that that $\sigma_0^2 \ne \sigma_1^2$, it is nonzero. So, $\phi$ is monotone with respect to $T$.

But, if the log likelihood ratio is monotone in $T$, then the likelihood ratio is monotone in $T$. So, the likelihood ratio is monotone in $\sum_{i = 1}^n(X_i - \mu)^2$.

**8. Suppose that $X_1, . . . , X_n$ form a random sample from the normal distribution with known mean $0$ and unknown variance $σ^2$, and suppose that it is desired to test the following hypotheses:**

$$H_0: σ^2 ≤ 2$$
$$H_1: σ^2 > 2$$

**Show that there exists a UMP test of these hypotheses at every level of significance $α_0$ $(0 < α_0 < 1)$.**

We demonstrated earlier that the likelihood ratio $L(\sigma_1^2) / L(\sigma_0^2)$ is monotone in the statistic $T = \sum_{i = 1}^n(X_i - \mu)^2$.

Since this is a one sided test, we can apply the UMP theorem to show that, for any significance level $\alpha_0$, exists an UMP test of the form,

$$
T \ge c
$$

Since, in this case, $\mu = 0$, we can simplify the expression for $T$.

$$
T = \sum_{i = 1}^n X_i^2
$$

**9. Show that the UMP test in Exercise 8 rejects $H_0$ when $\sum_{i = 1}^n X_i^2 \ge c$, and determine the value of $c$ when $n = 10$ and $α_0 = 0.05$.**



We have already shown that the UMP test is, for some $c$,

$$
\sum_{i = 1}^n X_i^2 \ge c
$$

We now consider the case where $n = 10$ and $\alpha_0 = 0.05$. We need to find $c$ such that,

$$
\sup_{\sigma^2 \le 2}\left(P\left(\sum_{i = 1}^n X_i^2 \ge c\right)\right) = 0.05
$$

We have that,

$$
X_i \sim \text{Normal}(0, \sigma^2)
$$

Consider the expression $X_i / \sigma$. This is a linear transformation of a normal variable, so the result is normal.

$$
E[X_i/\sigma] = {1 \over \sigma}E[X_i] = 0
$$

By the properties of variance,

$$
var(X_i / \sigma) = {1 \over \sigma^2}var(X_i) = {1 \over \sigma^2}\sigma^2 = 1
$$

So,

$$
{X_i \over \sigma} \sim \text{Normal}(0, 1)
$$

So,

$$
{X_i^2 \over \sigma^2} \sim \chi^2(1)
$$

So,

$$
\sum_{i = 1}^n {X_i^2 \over \sigma^2} \sim \chi^2(n)
$$

$$
{1 \over \sigma^2} \sum_{i = 1}^n X_i^2 \sim \chi^2(n)
$$

Let $\Phi_n$ be the cdf of the $\chi^2$ distribution with $n$ degrees of freedom. Let $\Phi_n^{-1}$ be the corresponding inverse function. Then,

$$
P({1 \over \sigma^2} \sum_{i = 1}^n X_i^2 \ge c_0) = 1 - \Phi_n(c_0)
$$

Then, since variance is positive,

$$
P(\sum_{i = 1}^n X_i^2 \ge \sigma^2 c_0) = 1 - \Phi_n(c_0)
$$

Let $c = \sigma^2 c_0$. Then,

$$
P(\sum_{i = 1}^n X_i^2 \ge \sigma^2 c) = 1 - \Phi_n(c / \sigma^2)
$$

Then,

$$
\sup_{\sigma^2 \le 2}(P(\sum_{i = 1}^n X_i^2 \ge \sigma^2 c)) = \sup_{\sigma^2 \le 2}(1 - \Phi_n(c / \sigma^2))
$$

So, we need to maximize $1 - \Phi_n(c / \sigma^2)$ subject to $\sigma^2 \le 2$. The derivative of $\Phi_n$ is positive on its positive domain. So, for some $k \gt 0$,

$$
{d \over d\sigma^2} (1 - \Phi_n(c / \sigma^2)) = -k{d \over d \sigma^2}(c\sigma^{-2})
$$

Then,

$$
{d \over d\sigma^2} (1 - \Phi_n(c / \sigma^2)) = -kc(-1)\sigma^{-4}
$$

$$
{d \over d\sigma^2} (1 - \Phi_n(c / \sigma^2)) = kc\sigma^{-4}
$$

So, when $c \ne 0$ (which we can reasonably assume), the derivative is always positive. So, the quantity is maximized at the upper bound on $\sigma^2$.

$$
\sup_{\sigma^2 \le 2}(1 - \Phi_n(c / \sigma^2)) = (1 - \Phi_n(c / 2))
$$

So,

$$
\sup_{\sigma^2 \le 2}(P(\sum_{i = 1}^n X_i^2 \ge \sigma^2 c)) = (1 - \Phi_n(c / 2))
$$

So, by the definition of $\alpha$,

$$
\alpha = 1 - \Phi_n(c / 2)
$$

We are interested in the case when $\alpha = 0.05$.

$$
1 - \Phi_n(c / 2) = 0.05
$$

Then,

$$
\Phi_n(c / 2) = 0.95
$$

$$
c/2 = \Phi_n^{-1}(0.95)
$$

$$
c = 2\Phi_n^{-1}(0.95)
$$

This can be solved numerically for the case where $n = 10$.

In [2]:
c = 2*stats.chi2(df = 10).ppf(0.95)
c

np.float64(36.61407610655029)

So,

$$
c = 36.61407610655029
$$

Our test is,

$$
\sum_{i = 1}^n X_i^2 \ge 36.61407610655029
$$

**13. Suppose that four observations are taken at random from the normal distribution with unknown mean μ and known variance 1. Suppose also that the following hypotheses are to be tested:**

$$H_0: μ ≥ 10$$
$$H_1: μ < 10$$

**a. Determine a UMP test at the level of significance $α_0 = 0.1$.**

We have that,

$$
P(X_1...X_n) = \prod_{i = 1}^n (2\pi)^{-1/2} \exp(-(x_i - \mu)^2/2)
$$

So, the likelihood of $\mu$ is given by,

$$
L(\mu) = \prod_{i = 1}^n (2\pi)^{-1/2} \exp(-(X_i - \mu)^2/2)
$$

$$
L(\mu) = (2\pi)^{-n/2} \prod_{i = 1}^n  \exp(-(X_i - \mu)^2/2)
$$

$$
L(\mu) = (2\pi)^{-n/2} \exp \left( -{1 \over 2} \sum_{i = 1}^n (X_i - \mu)^2 \right)
$$

The likelihood ratio, $\phi$, between $\mu_1$ and $\mu_0$ is given by,

$$
\phi = {
    (2\pi)^{-n/2} \exp \left( -{1 \over 2} \sum_{i = 1}^n (X_i - \mu_1)^2 \right) \over
    (2\pi)^{-n/2} \exp \left( -{1 \over 2} \sum_{i = 1}^n (X_i - \mu_0)^2 \right)
}
$$

$$
\phi = {
    \exp \left( -{1 \over 2} \sum_{i = 1}^n (X_i - \mu_1)^2 \right) \over
    \exp \left( -{1 \over 2} \sum_{i = 1}^n (X_i - \mu_0)^2 \right)
}
$$

$$
\phi = {
    \exp \left( -{1 \over 2} \sum_{i = 1}^n (X_i - \mu_1)^2 - \left( -{1 \over 2} \sum_{i = 1}^n (X_i - \mu_0)^2 \right) \right)
}
$$

$$
\phi = {
    \exp \left( -{1 \over 2} \sum_{i = 1}^n \left( (X_i - \mu_1)^2 - (X_i - \mu_0)^2 \right) \right)
}
$$

$$
\phi = {
    \exp \left( -{1 \over 2} \sum_{i = 1}^n \left( (X_i^2 - 2X_i \mu_1 + \mu_1^2) - (X_i^2 - 2X_i \mu_0 + \mu_0^2) \right) \right)
}
$$



$$
\phi = {
    \exp \left( -{1 \over 2} \sum_{i = 1}^n \left( - 2X_i \mu_1 + \mu_1^2 + 2X_i \mu_0 - \mu_0^2 \right) \right)
}
$$

$$
\phi = {
    \exp \left( -{1 \over 2} \sum_{i = 1}^n \left( 2(\mu_0 - \mu_1)X_i + (\mu_1^2 - \mu_0^2) \right) \right)
}
$$

$$
\phi = {
    \exp \left( -{1 \over 2} \left( n(\mu_1^2 - \mu_0^2) + 2(\mu_0 - \mu_1) \sum_{i = 1}^n X_i  \right) \right)
}
$$

$$
\phi = {
    \exp \left(  (\mu_1 - \mu_0 ) \left( \sum_{i = 1}^n X_i \right) - {n(\mu_1^2 - \mu_0^2) \over 2} \right)
}
$$

Let,

$$
T = \sum_{i = 1}^n X_i
$$

Then,

$$
\phi = {
    \exp \left(  (\mu_1 - \mu_0) T \right) \exp\left(-{n(\mu_1^2 - \mu_0^2) \over 2} \right)
}
$$

If $\mu_1 \gt \mu_0$, then $\mu_1 - \mu_0 \gt 0$. So, the likelihood ratio is monotone increasing in $T$, and the null is a lower bound in $\mu$. So, a UMP test for any $\alpha$ is given by,

$$
T \le c
$$

Consider the distribution of $T$. $T$ is a linear combination of normal variables, so it's normally distributed.

Take the expectation.

$$
E[T] = E[\sum_{i = 1}^n X_i]
$$

$$
E[T] = \sum_{i = 1}^n E[X_i]
$$

$$
E[T] = \sum_{i = 1}^n \mu
$$

$$
E[T] = n \mu
$$

Take the variance.

$$
var(T) = var(\sum_{i = 1}^n X_i)
$$

The $X_i$ are independent. So,

$$
var(T) = \sum_{i = 1}^n var(X_i)
$$

$$
var(T) = \sum_{i = 1}^n 1
$$

$$
var(T) = n
$$

So,

$$
T \sim \text{Normal}(n\mu, n)
$$

Let $Z$ be a standard normal variable. Consider the quantity $n\mu + \sqrt{n}Z$. This is a linear transformation of a normal variable, so it's normal.

By the properties of expectation,

$$
E[n\mu + \sqrt{n}Z] = n\mu + \sqrt{n}E[Z] = n\mu
$$


By the properties of variance,
$$
var[n\mu + \sqrt{n}Z] = n var[Z] = n
$$

So,

$$
n\mu + \sqrt{n}Z \sim \text{Normal}(n\mu, n)
$$

So, $n\mu + \sqrt{n}Z$ is equal in distribution to $T$.

So,

$$
P(T \le c) = P(n\mu + \sqrt{n}Z \le c)
$$

$$
P(T \le c) = P(Z \le {c - n\mu \over \sqrt{n}})
$$

Let $\Phi$ denote the cdf of the standard normal distribution. Let $\Phi^{-1}$ be the corresponding inverse function. Then,

$$
P(T \le c) = \Phi({c - n\mu \over \sqrt{n}})
$$

By the definition of $\alpha$,

$$
\alpha = \sup_{H_0} P(T \le c)
$$

So,

$$
\alpha = \sup_{\mu \ge 10} \Phi({c - n\mu \over \sqrt{n}})
$$

The derivative of $\Phi$ is positive and $n$ is positive. So, this is maximized when $\mu$ is small. In this case, that means $\mu$ is at the lower bound, $\mu = 10$. So,

$$
\alpha = \Phi({c - n10 \over \sqrt{n}})
$$

We are interested in the case where $\alpha = 0.1$. So,

$$
\Phi({c - n10 \over \sqrt{n}}) = 0.1
$$

$$
{c - n10 \over \sqrt{n}} = \Phi^{-1}(0.1)
$$

$$
c = \Phi^{-1}(0.1)\sqrt{n} + 10n
$$

This can be solved numerically for the case where $n = 4$.

In [3]:
c = stats.norm(0, 1).ppf(0.1)*(4**0.5) + 10*4
c

np.float64(37.4368968689108)

So, in this case,

$$
c = 37.4368968689108
$$

So, our test is,

$$
\sum_{i = 1}^n X_i \lt 37.4368968689108
$$

**b. Determine the power of this test when $μ = 9$.**

We are interested in the probability of rejecting when $\mu = 9$.

Earlier, we determined that,

$$
P(T \le c) = \Phi({c - n\mu \over \sqrt{n}})
$$

So,

$$
P(T \le c | \mu = 9) = \Phi({c - 9n \over \sqrt{n}})
$$

This can be solved numerically for the case when $n = 4$.

In [4]:
stats.norm(0, 1).cdf((37.4368968689108 - 9*4)/(4**0.5))

np.float64(0.7637595841058833)

So, when $\mu = 9$, we have a probability of 0.7637595841058833 of rejecting the null.

**c. Determine the probability of not rejecting H0 if $μ = 11$.**

The probability of rejecting is given by,

$$
P(T \le c | \mu = 11) = \Phi({c - 11(4) \over \sqrt{4}})
$$

So, the probability of not rejecting is,

$$
P(T \gt c | \mu = 11) = 1 - \Phi({c - 44 \over 2})
$$

This can be solved numerically.

In [5]:
1 - stats.norm(0, 1).cdf((37.4368968689108 - 44)/2)

np.float64(0.9994838117703562)

The probability of not rejecting the null when $\mu = 11$ is 0.9994838117703562.